tvx/extract_msg.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/ipfs/go-cid"
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/oni/tvx/lotus"
+	"github.com/filecoin-project/oni/tvx/state"
+)
+
+var extractMsgCmd = &cli.Command{
+	Name:        "extract-message",
+	Description: "generate a message-class test vector by extracting it from a network",
+	Flags:       []cli.Flag{&cidFlag, &apiFlag, &fileFlag},
+	Action:      runExtractMsg,
+}
+
+func runExtractMsg(c *cli.Context) error {
+	_ = os.Setenv("LOTUS_DISABLE_VM_BUF", "iknowitsabadidea")
```

Comment:
needed? comment of what this changes would be valuable

Reply:
Adding.

Reply:
https://github.com/filecoin-project/oni/pull/177/commits/4bb157304cceea565abffdeab570746261108732